### DISCUSSION TITLE
A quick note

### DISCUSSION THREAD

**Discussion by rmarquis** [etlegacy/etlegacy#2012] [General]
Hello there,

My apologies for my silence and absence the past few months. I have retired from the project some time ago, but didn't find the time to say more. So here is the quick note to officially announce it.

At least some of you know already that I have been extremely busy recently (new jobs, new city, long commuting time) to a point my free time dropped to an absurd level. I've also been away from Discord for even longer for many reasons, which honestly was an extremely positive decision, time and energy wise.

I haven't really followed recent news and development, but I've heard from one of you that Steam has become a reality. I've always been a proponent of the idea that an open source project belongs to the people that push it forward (rather to those that complain about it - doers vs sayers), and I'm glad to see that new faces actively joined the effort (ryzyk, Acid, Helldembez, ...) over the months. Thanks to them, and also to the other long standing members of the projects.

I shouldn't have access to any of the infrastructure anymore (GitHub, game server, website, admin panel, ..), but don't hesitate to remove me from any left over if you notice it. There's still a few things I'd like to do for the project, but when/if I find the time I'll push a PR as an external contributor.

Thanks for reading. The past 7 years have been a wild ride with lot of up and down among a crazy community of players, but I'm glad to see the project still moving forward. Keep up the hard work!

Feel send any question, suggestion or insult my way, I'll do my best to answer back or send bags of dicks!

**Comment by realkemon** [comment]
Godspeed, my friend!
Thank you for everything you have done over the years.

**Comment by ryzyk-krzysiek** [comment]
Thank you for all your work on ET Legacy project. Without people like you that kept working on it for so many years I wouldn't even be here right now trying to help improve it myself, so it is greatly appreciated. Looking forward to your future PRs and good luck to you.

**Comment by mittermichal** [comment]
Thanks for all the work on the project. I wish you more free time. Use long commute time to read or to listen to something :D
I hope to quit discord one day too, I don't have it on phone for a few months already.
